frontend-react/src/App.tsx

```diff
@@ -225,6 +226,11 @@ const App = () => {
                                     authorize={PERMISSIONS.PRIME_ADMIN}
                                     component={AdminLastMileFailures}
                                 />
+                                <AuthorizedRoute
+                                    path="/admin/senddash"
```

Comment:
another "case" where there are two conventions going. Routes seem to be using both kebab case and flat case. I'm pretty hardcore in favor of kebab case here, but we can wait and discuss tomorrow at eng sync?

Reply:
So for renaming the url path on our site, I'm happy to change it to whatever we decide is should be. 
`/admin/send-dash` (kebab)
`/admin/send_dash` (snake)
`/admin/sendDash` (camel)

If it's related to the API call, suspect it will be punted to the next API working group meeting.

I've started a list of API inconsistencies here #6243, you should add your api naming convention preference to it. I suspect we're in the the situation we're in now because nobody had a strong enough opinion ... and naming is hard.